feature/system/tests/system_base_test/README.md

```diff
@@ -37,3 +37,19 @@ used for mTLS.
 2. Each test will then create a client to those services and valid each service is properly
    listening on the standard port.
 3. Validate client properly connects and execute a simple RPC to validate no errors are returned.
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths intended to be covered by this test. OC
+paths used for test setup are not listed here.
+
+```yaml
+paths:
+  ## Config paths:
+  ## State paths: N/A
+
+rpcs:
+  gnmi:
+    gNMI.Set:
```

Comment:
Please include the list of all get , set, state and configs OCs used under all the .go files here

Reply:
Hi Swetha,
Updated the OC paths in the yaml block which are used in go files.